src/runtime/manifest.ts

```diff
@@ -136,8 +136,21 @@ interface ManifestLoadOptions {
 }
 
 export class Manifest {
+  public static readonly canonical = `
```

Comment:
It would be great if this was in its own file. 

It can be a typescript file if that makes things significantly easier, but I wouldn't add it to this already very long file.

Reply:
moved to its own file, keeping it .ts to avoid dealing with loader etc.
i think it'd be nicer to add it to the `options` param of the `parse` method, but then it'll always be the default for now, so i'll just do it later...